test/models/movie_test.rb

```diff
@@ -0,0 +1,15 @@
+require "test_helper"
+
+describe Movie do
+
+  it "a movie object can be saved to the database" do
```

Comment:
negative tests?  What about when the inventory, or available inventory is 0.  Can you create a movie with negative inventory?